### PR TITLE
Assume initial mix of shell efficiencies

### PR DESCRIPTION
**Pull request by robinhasse**
- until now, we assumed all historic buildings to have `"low"` efficiency building shell
- this PR assumes a mix of all three classes instead
- I rescaled the UE demand of the classes such that the mean is maintained
- references of GDP/POP in USD2005 did not work anymore and are fixed now
- I removed all instances of `brick::getBrickMapping` such that mappings are tracked by madrat

### CODE REVIEW

File: R/calcUEdemand.R [hunk 1]
```diff
@@ -35,14 +37,20 @@ calcUEdemand <- function() {
     group_by(across(-all_of(c("value")))) %>%
     summarise(value = mean(.data[["value"]]), .groups = "drop")
 
+  # scale demand such that average is maintained with differentiated efficiency
```
Comment (ricardarosemann) [inline R/calcUEdemand.R:40]: Here it was not clear to me with respect to what you want to maintain the average. I guess it's with respect to the previous setting where the entire stock has ```bs = low``` with relative demand 1? Then you can simply write: Scale demand such that the average relative demand is 1.

File: R/calcUValue.R [hunk 2]
```diff
@@ -18,7 +17,7 @@ calcUValue <- function() {
     select("vin", "vinHotmaps")
 
   # map Hotmaps building types
-  typMap <- getBrickMapping("buildingType.csv") %>%
+  typMap <- toolGetMapping("buildingType.csv", "sectoral", "brick") %>%
```
Comment (ricardarosemann) [inline R/calcUValue.R:20]: See above :-)

**Review by ricardarosemann — APPROVED** [review]
Thanks!
Some minor comments; I suggest to state the optional arguments of toolGetMapping explicitly and I tried to mark every occurrence of this, that's why there are so many comments.

**Comment by robinhasse** [comment]
> I suggest to state the optional arguments of toolGetMapping explicitly

Did that. Thanks!

**Pull request merged**